starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/GroceryViewHolder.java

```diff
@@ -0,0 +1,29 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.TextView;
+
+/**
+ * Created by ds on 10/25/16.
+ */
+
+public class GroceryViewHolder extends RecyclerView.ViewHolder {
+
+    private TextView mFoodNameTV;
```

Comment:
A `ViewHolder` is one of the few places where people usually _don't_ make the variables private, and _don't_ bother with getters and setters. Instead, you can make this variable `public` or just not specify an access modifier, in which case the access defaults to "package private", meaning any code in files in the same folder can access/modify it.